terraform/github/repositories/ministryofjustice/operations-engineering-kpi-dashboard.tf

```diff
@@ -1,9 +1,9 @@
-module "operations-engineering-kpi-dashboard-poc" {
```

Comment:
I believe you may need a [moved](https://drive.google.com/drive/folders/1PiadCKFPQPL9gYyZRhsc-9Kk70M90-nB) block for this

At the moment, the plan is going to destroy all the resources for `operations-engineering-kpi-dashboard-poc` and create a set of new resources fo `operations-engineering-kpi-dashboard` - using the move block should mean the resources are kept the same 😁

Reply:
🔥 is good right?